pkgs/applications/blockchains/solana-validator/default.nix

```diff
@@ -44,6 +47,8 @@ let
   version = pinData.version;
   hash = pinData.hash;
   cargoHash = pinData.cargoHash;
```

Comment:
Unused bind

Reply:
nice catch. this line should be removed then i will merge this unless there are any objections.